src/test_l1_dsVideoPort.c

```diff
@@ -2980,53 +2809,44 @@ void test_l1_dsVideoPort_negative_dsSetForceDisable4KSupport(void) {
  * |05|Compare the array values and make sure they are equal | | dsERR_NONE | The values must be equal |
  * |06|Call dsVideoPortTerm() - Terminate the video port system | | dsERR_NONE | Termination must be successful | 
  * 
  */
 void test_l1_dsVideoPort_positive_dsGetForceDisable4KSupport(void) {
-	gTestID = 45;
-	UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
-
-	dsError_t status;
-	intptr_t handle[NUM_OF_PORTS]={HANDLE_ARRAY_INIT};
-
-	bool disable4KArray1[NUM_OF_PORTS];
-	bool disable4KArray2[NUM_OF_PORTS];
-
-	// Step 01: Initialize video port system
-	status = dsVideoPortInit();
-	UT_ASSERT_EQUAL(status, dsERR_NONE);
-
-	// Step 02: Get the video port handle
-	for (int i = 0; i < NUM_OF_PORTS; i++) {
-		status = dsGetVideoPort(kPorts[i].id.type, kPorts[i].id.index, &(handle[i]));
-		UT_ASSERT_EQUAL(status, dsERR_NONE);
-	}
-
-	// Step 03: Retrieve whether 4K support is disabled
-	for (int i = 0; i < NUM_OF_PORTS; i++) {  
-		status = dsGetForceDisable4KSupport(handle[i], &disable4KArray1[i]);
-		UT_ASSERT_EQUAL(status, dsERR_NONE);
-	}
-
-	// Step 04: Repeat the retrieval
-	for (int i = 0; i < NUM_OF_PORTS; i++) {
-		status = dsGetForceDisable4KSupport(handle[i], &disable4KArray2[i]);
-		UT_ASSERT_EQUAL(status, dsERR_NONE);
-	}
-
-	// Step 05: Compare the array values
-	for (int i = 0; i < NUM_OF_PORTS; i++) {
-		UT_ASSERT_EQUAL(disable4KArray1[i], disable4KArray2[i]);
-	}
-
-	// Step 06: Terminate the video port system
-	status = dsVideoPortTerm();
-	UT_ASSERT_EQUAL(status, dsERR_NONE);
-
-	UT_LOG("\n Out %s\n", __FUNCTION__); 
+    gTestID = 45;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t status;
+    intptr_t handle = 0;
+
+    bool disable4K1 = false;
+    bool disable4K2 = false;
+
+    // Step 01: Initialize video port system
+    status = dsVideoPortInit();
+    UT_ASSERT_EQUAL_FATAL(status, dsERR_NONE);
+
+    // Step 02: Get the video port handle
+    for (int i = 0; i < gDSvideoPort_NumberOfPorts; i++) {
+        status = dsGetVideoPort(gDSVideoPortConfiguration[i].typeid, gDSVideoPortConfiguration[i].index, &(handle));
+        UT_ASSERT_EQUAL(status, dsERR_NONE);
+        // Step 03: Retrieve whether 4K support is disabled
+        status = dsGetForceDisable4KSupport(handle, &disable4K1);
+        UT_ASSERT_EQUAL(status, dsERR_OPERATION_NOT_SUPPORTED);
+        // Step 04: Repeat the retrieval
+        status = dsGetForceDisable4KSupport(handle, &disable4K2);
+        UT_ASSERT_EQUAL(status, dsERR_OPERATION_NOT_SUPPORTED);
+        // Step 05: Compare the array values
+        UT_ASSERT_EQUAL(disable4K1, disable4K2);
```

Comment:
i think we may need to compare 
this also we take care in code clan up activity